app/javascript/application.js

```diff
@@ -1,7 +1,6 @@
 // Configure your import map in config/importmap.rb. Read more: https://github.com/rails/importmap-rails
-import "@hotwired/turbo-rails"
-import "controllers"
+import "@hotwired/turbo-rails";
+import "controllers";
```

Comment:
Not that I have any _problem_ with these...but my eyebrows are raised.

Reply:
Oh, I agree, I don't like semicolons. I removed these at some other point but my have reverted them back in.